SpatialGDK/Source/SpatialGDK/Private/Utils/ComponentReader.cpp

```diff
@@ -265,7 +265,7 @@ void ComponentReader::ApplyProperty(Schema_Object* Object, Schema_FieldId FieldI
 
 		ReadStructProperty(ValueDataReader, StructProperty, NetDriver, Data, bHasUnmapped);
 
-		if (bHasUnmapped)
```

Comment:
This was the only place the bHasUnmapped flag was checked. Should we remove it from the various functions signatures ? It's unclear what it is used for.

Reply:
How about instead we modify `USpatialPackageMapClient::SerializeObject` to return false if the Obj failed to resolve on read? This would match closer to what Unreal does.

Reply:
Done. I had to pull SpatialNetBitReader in SpatialPackageMapClient though.